frontend/pages/packs/EditPackPage/_styles.scss

```diff
@@ -1,4 +1,17 @@
 .edit-pack-page {
+  &__back-link {
+    font-size: $x-small;
+    color: $core-vibrant-blue;
+    font-weight: $bold;
+    text-decoration: none;
```

Comment:
This is the 7th time in the code base we're defining the `&__back-link` style. But, every other time it's page-specific, so it's necessary to create it again here.

This is a great example of how our current approach to styling pages isn't scaling.